src/LuaPiGui.cpp

```diff
@@ -423,11 +423,30 @@ static int l_pigui_begin(lua_State *l)
 	return 1;
 }
 
+/*
+ * Function: columns
+ *
+ * Initiates drawing of a table with n columns
+ *
+ * > local ui = require 'pigui'
+ * > ui.columns(n_columns, label, has_border)
+ * > ui.text("row 1, column 1")
+ * > ui.nextColumn()
+ * > ui.text("row 1, column 2")
+ * > ui.columns(1)
+ *
+ * Parameters:
+ *
+ *   n_columns - integer, the number of columns,
+ *   label - optional string, the label of the object
+ *   border - optional, bool, draw vertical column border or not (default)
```

Comment:
@impaktor API documentation should use strong, clear, and concise language. Please prefer something like the below to document this parameter:
```suggestion
 *   border - optional bool, draws a vertical border between columns. Defaults to false.
```